exercises/practice/state-of-tic-tac-toe/.meta/config.json

```
@@ -0,0 +1,17 @@
+{
+  "authors": ["AbhilashJN"],
+  "files": {
+    "solution": [
+      "state_of_tic_tac_toe.go"
+    ],
+    "test": [
+      "state_of_tic_tac_toe_test.go"
+    ],
+    "example": [
+      ".meta/example.go"
+    ]
+  },
+  "blurb": "Determine the game state of a match of Tic Tac Toe.",
+  "source": "Created by Sascha Mann for the Julia track of the Exercism Research Experiment.",
+  "source_url": "https://github.com/exercism/research_experiment_1/tree/julia-dev/exercises/julia-1-a"
```

```suggestion
  "source": "Created by Sascha Mann, Jeremy Walker, and BethanyG originally for the Julia track on Exercism.",
  "source_url": "https://github.com/exercism/julia/pull/413"
```

Changed it to be similar to what the original exercise states as source: https://github.com/exercism/julia/blob/da8b107b0d8b92e8eea1ad7188d9699b0fb7013d/exercises/practice/killer-sudoku-helper/.meta/config.json#L21-L22

---

Oh, ok. Disregard my suggestion then and let's leave the file as is.